app/models/config.rb

```diff
@@ -252,4 +268,13 @@ def validate_patient_archive
     def validate_shared_reset
       validate_number && options.last.to_i.between?(SHARED_MIN_DAYS, SHARED_MAX_DAYS)
     end
+
+    def validate_length
+      total_length = 0
+      options.each do |option|
+        total_length += option.length
```

Comment:
I probably gave unclear directions here, so please forgive me. There are legit reasons for configs to have a total length of 1k+ - for example, at least two funds have a very long list of funds they work with that come out to 1500+ characters. So I think we'll need to adjust the strategy here in particular. We could:
* raise the cap to 4k, like notes are at, and keep this strategy as you have it written here (def fixes it, still keeps us flexible)
* check individual elements against the 150 char limit (mostly fixes the root problem, keeps us flexible)
* say to ourselves 'configs are an admin-only endpoint and so they don't really need validation' (a reasonable approach, would leave only this admin-only module vulnerable to this screwing-around-with)

I'll defer to you on the right path here.

Reply:
Makes sense! I opted for the first option, raising the cap to 4k. I don't feel strongly, but overall I feel like it gives sufficient protection while minimizing the likelihood of false positives compared to a per-element limit.